packages/react-strict-dom/tests/__snapshots__/html-test.native.js.snap-native

```diff
@@ -1554,6 +1554,38 @@ exports[`<html.*> style polyfills inherited styles 1`] = `
 </View>
 `;
 
+exports[`<html.*> style polyfills inherited styles for auto-fix of raw strings 1`] = `
+<View
+  experimental_layoutConformance="strict"
+  style={
+    {
+      "position": "static",
+    }
+  }
+>
+  <View
+    experimental_layoutConformance="strict"
+    style={
+      {
+        "position": "static",
+      }
+    }
+  >
+    <Text
+      style={
+        {
+          "color": "green",
+          "fontSize": 64,
+          "lineHeight": 192,
```

Comment:
This matches the computed styles that the fixture code generates in a web browser.